website/blog/flink-native-support/index.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Data lineage for Apache Flink
+date: 2024-11-15
+authors: [Leszczynski]
+description: Flink native lineage support will bring OpenLineage integration to the new era.
+---
+Apache Flink is a powerful stream processing engine widely adopted in the industry.
+Flink jobs are capable of emitting OpenLineage events for already two years.
+However, it is a recent joint effort of Flink and OpenLineage communities that will bring this integration into the new era.
+
+<!--truncate-->
+
+### Existing OpenLineage Flink integration
+
+As of November 2024, [OpenLineage Flink integration](https://github.com/OpenLineage/OpenLineage/tree/main/integration/flink) supports Flink versions `1.15` to `1.18` and emits OpenLineage events complying with the dataset naming model.
+It supports Kafka sources and sinks with Avro and Protobuf, JDBC and Cassandra connectors with extracting lineage from the queries executed.
```

Comment:
quite a lot of `and`'s and `with`'s and commas for one sentence

I don't have good suggestion on how to rephrase that yet